### ISSUE TITLE
The example only works when you are not usign PKCE

### ISSUE THREAD

**Issue by AgusMarquezUala** [oktadeveloper/okta-nextjs-typescript-example#2]
The OktaProvider should support PKCE, since its the most secure way to implement an OpenID-OAuth flow for an SPA. The only way to implement that using nextAuth is by setting a custom provider.

**Comment by sebastianmulders** [comment]
@AgusMarquezUala good catch. Yet we still can't get it to work here, even with a custom provider. Every time the session doesn't seem to get set properly. Did you get it to work? If yes, do you mind sharing your solution?
